MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: Description in .NET MAUI Step ProgressBar control | Syncfusion
+description: Learn here all about Description support in Syncfusion .NET MAUI Step ProgressBar (SfStepProgressBar) control and more.
+platform: maui
+control: SfStepProgressBar
+documentation: ug
+---
+
+# Description in .NET MAUI StepProgressBar (Step Progress Bar)
+Each step in a multi-step process has a different operation. To display information about a step's status can be shown on either side. A primary description will be on the right or bottom of the step, and a secondary description will be on the left or top of the step.
+
+## StepProgressBar overview
+The following overview image illustrates the major elements presented in StepProgressBar.
+
+## Text 
+The primary and secondary description for a step view can be set using the [PrimaryText]() and [SecondaryText]() properties as demonstrated in the following code example.
+
+* Create a simple Observable Collection of item for the step progressbar,
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="25 26 27 28" %}
+
+public class ViewModel
+{
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    private ObservableCollection<StepProgressBarItem> stepProgressItem;
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        stepProgressItem = new ObservableCollection<StepProgressBarItem>();
+        stepProgressItem.Add(new StepProgressBarItem() { PrimaryText = "Completed", SecondaryText = "Task" });
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to step progressbar
+
+The Step progressbar control allows you to bind item collection by setting the [ItemsSource]() property from the [SfStepProgressBar](). Bind item collection in both XAML and C#.
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="6" %}
+
```

Comment:
Binding related and view model related already explained in Populating Items topic and check and optimize

Reply:
Need to add view model class to show how to give the primary and secondary text. I have removed the item source description. Because it added in Populating Items topic.